dist/libs/deso-protocol/src/index.d.ts

```diff
@@ -11,7 +11,7 @@ import { Node } from './lib/Node/Node';
 import { Wallet } from './lib/wallet/Wallet';
 import { Transactions } from './lib/transaction/Transaction';
 export declare class Deso {
-    constructor();
+    constructor(nodeUri?: string, identityConfig?: IdentityConfig);
```

Comment:
Lets makes this a single param also so 
export interface DesoConfig {nodeUri?: string, identityConfig: IdentityConfig}